src/apiCalls.js

```diff
@@ -1,6 +1,6 @@
 
-const searchFetch = (input, genre) => {
-  return fetch(`https://squadfinder2205be.herokuapp.com/api/v1/search?search=${input}&genres=${genre}`)
+const searchFetch = (input) => {
+  return fetch(`https://squadfinder2205be.herokuapp.com/api/v1/search?search=${input}`)
     .then(response => response.json())
 }
 
```

Comment:
Since the genres will be handled on the FE now, I took genres out of the API call.